src/sentry/spans/consumers/process/factory.py

```diff
@@ -86,6 +85,7 @@ def _process_message(message: Message[KafkaPayload]) -> KafkaPayload:
     payload = msgpack.unpackb(message.payload.value)
     relay_span = payload["span"]
     relay_span["project_id"] = payload["project_id"]
+    relay_span["event_id"] = payload.get("event_id", "")
```

Comment:
I noticed the discussion on trace ID. The event_id will be in the same format, see [test case here](https://github.com/getsentry/relay/blob/7b39563ef64e27d7c76950d1a17608d020c32bb6/tests/integration/test_store.py#L1213-L1222).